### ISSUE TITLE
Volume Control for HDMI 2 ARC connected soundbar

### ISSUE THREAD

**Issue by ds2288** [AntonioMeireles/homebridge-vieramatic#102]
It’d be great for the volume “fan”option to have the ability to control the home cinema sound via ARC, the same way the TV remote does. I see it can only control if you are using TV sound.

**Comment by AntonioMeireles** [comment]
Hi, 

interesting. No gear at home to quickly reproduce/fix.

quick question - **if** you increase volume thru the hardware +/- keys on iphone remote (after selecting your TV) volume goes up/down too when via ARC ?  

Thanks in advance

**Comment by ds2288** [comment]
No, the TV states that it won’t work because the sounds is not set to TV (it’s set to Home Cinema).
Disregard anyway, the plugin seems to have crashed overnight, utilising 4.0.35, because 4.0.36 won’t start in Homebridge, a log full of errors. I’ve had to delete your plugin as it was causing issues amongst my setup.

**Comment by AntonioMeireles** [comment]
@ds2288 

i'd love your crash logs of .36 ... i've had another report of issues with it but need more pointers...

**Comment by ds2288** [comment]
If I have time next week I’ll set it up again and send logs.

Regards,

David Sammut
________________________________

**Comment by AntonioMeireles** [comment]
@ds2288 

regarding the crashes you've had with 4.0.36. can you try please just published 4.0.37, and see if it behaves as it should (i.e, stably) on your side ?

thanks in advance!

**Comment by AntonioMeireles** [comment]
@ds2288 

Thanks for the update! 

Now, the million dollar question - Volume control works via ARC when using Panasonic's mobile app ([IOS install link](https://apps.apple.com/us/app/panasonic-tv-remote-3/id1435893441) and [Android install link](https://play.google.com/store/apps/details?id=com.panasonic.ais_pfdc_tvremote3_gbl&hl=en)) ? Could you check ?

If it does indeed work, then there's a way to do it via my plugin. and we'll find it one way or another ... if not, ball in Panasonic's court...

 Thanks again!

**Comment by ds2288** [comment]
Not sure I can help with that. That app has never recognised my TV thus I’ve never been able to set it up. The same just happened again. It doesn’t find the TV on the network. It’s a pretty old app.

**Comment by UltraBuk** [comment]
The issue is already a few days old, but I am currently in the same situation. The volume control via the remote app works with a ARC connected soundbar.
